components/raftstore/src/coprocessor/mod.rs

```diff
@@ -203,6 +203,18 @@ pub trait SplitCheckObserver<E>: Coprocessor {
     );
 }
 
+#[derive(Debug, Default)]
+pub struct StoreSizeInfo {
+    pub capacity: u64,
```

Comment:
Please add some comments.

Reply:
And what's the meaning of all these fields? Why there is a `used` when the other two are available?